client/my-sites/plans-features-main/jetpack-faq.jsx

```diff
@@ -35,13 +35,11 @@ const JetpackFAQ = () => {
 	return (
 		<FAQ>
 			<FAQItem
-				question={ translate( 'What is the cancellation policy?' ) }
+				question={ translate( 'What is your cancellation policy?' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.